hera_mc/geo_handling.py

```diff
@@ -286,18 +272,32 @@ def print_loc_info(self, loc_list, verbosity='h'):
             print("No locations found.")
             return False
         for a in loc_list:
-            if verbosity == 'm' or verbosity == 'h':
-                print('station_name: ', a.station_name)
-                print('\teasting: ', a.easting)
-                print('\tnorthing: ', a.northing)
-                print('\tlon/lat:  ', a.lon, a.lat)
-                print('\televation: ', a.elevation)
-                print('\tstation description ({}):  {}'.format(a.station_type, a.desc))
-                print('\tcreated:  ', cm_utils.get_time_for_display(a.created_date))
-            elif verbosity == 'l':
-                print(a)
+            print('station_name: ', a.station_name)
+            print('\teasting: ', a.easting)
+            print('\tnorthing: ', a.northing)
+            print('\tlon/lat:  ', a.lon, a.lat)
+            print('\televation: ', a.elevation)
+            print('\tstation description ({}):  {}'.format(a.station_type_name, a.desc))
+            print('\tcreated:  ', cm_utils.get_time_for_display(a.created_date))
         return True
 
+    def parse_station_types_to_check(self, sttc, add_stations):
+        self.get_station_types(add_stations=add_stations)
+        if isinstance(sttc, (str, unicode)):
+            if sttc.lower() == 'all':
+                return self.station_types.keys()
+            else:
+                sttc = [sttc]
+        sttypes = []
+        for s in sttc:
+            if s.lower() in self.station_types.keys():
+                sttypes.append(s.lower())
+            else:
+                for k in self.station_types.keys():
+                    if s.upper() in self.station_types[k]['Prefix']:
+                        sttypes.append(k)
```

Comment:
This loop could be:

```python
for k, st in self.station_types.iteritems():
    if s.upper() in st['Prefix']:
        sttypes.append(k) # or sttypes.add(k) if sttypes becomes a set
```

Also, is this `s.upper() in self.station_types[k]['Prefix']` the desired behavior? I.e, I can pass an argument of `'H'` and it will match all station types whose prefixes include the letter `H`? A few example arguments in the docstring might be instructive.